### PR TITLE
Edited Amorphous Body mutation description

### PR DESCRIPTION
**Pull request by KayoKake**
#### Summary

SUMMARY: None

#### Purpose of change

Without the edit, the description appears to only suggest the mutation has a positive effect on players. The mutation adds a 1.25x movement modifier, making them slower, so I made the description reflect that.

### CODE REVIEW

**Comment by KayoKake** [comment]
First time doing this, don't shoot me too quickly for getting this stuff messed up

**Comment by AMurkin** [comment]
`Summary: None`
would be more appropriate here.

**Comment by KayoKake** [comment]
> `Summary: None`
> would be more appropriate here.

What's the point in not making a summary? Is it just too small of a change for it to be worth making a summary for?

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -5039,7 +5039,7 @@
     "points": -2,
     "visibility": 10,
     "ugliness": 10,
-    "description": "Your flesh is a pleasing gel-like consistency.  Your bodily functions seem to be moving around, and your leg-equivalents flow comfortably.",
+    "description": "Your flesh is a pleasing gel-like consistency.  Your bodily functions seem to be moving around, but your leg-equivalents flow slower.",
```
Comment (I-am-Erk) [inline data/json/mutations/mutations.json]:
```suggestion
    "description": "Your flesh is a pleasing gel-like consistency.  Your bodily functions seem to be moving around, and your leg-equivalents flow comfortably - if a little slower than your old meat-legs.",
```

In your edit it's unclear what they flow slower than.

**Comment by anothersimulacrum** [comment]
> >  `Summary: None`
> >  would be more appropriate here.
>
> What's the point in not making a summary? Is it just too small of a change for it to be worth making a summary for?

Yes

File: data/json/mutations/mutations.json (hunk 1 above)
Comment (KayoKake) [reply data/json/mutations/mutations.json]: Solid edit. You word it masterfully, impressive, I spent some time thinking of a good way to word it but yours is great.

**Comment by KayoKake** [comment]
[Was a review unnecessary? Was I supposed to leave it as it was?]

**Pull request merged**